public/bundle.js

```diff
@@ -94,7 +94,7 @@
gdXBkYXRlUXVhbnRpdHlDaG9pY2UoY2hvaWNlKSB7XHJcbiAgICB0aGlzLnNldFN0YXRlKHtxdWFudGl0eUNob2ljZTogTnVtYmVyKGNob2ljZSl9KTtcclxuICB9XHJcblxyXG4gIHVwZGF0ZVBlcnNvbmFsaXphdGlvbkNob2ljZShjaG9pY2UpIHtcclxuICAgIHRoaXMuc2V0U3RhdGUoe3BlcnNvbmFsaXphdGlvbkNob2ljZTogY2hvaWNlfSk7XHJcbiAgfVxyXG5cclxuICB1cGRhdGVTdGF0ZShuZXdEYXRhKSB7XHJcbiAgICB0aGlzLnNldFN0YXRlKG5ld0RhdGEpO1xyXG4gIH1cclxuXHJcbiAgY29tcG9uZW50RGlkTW91bnQoKSB7XHJcbiAgICB0aGlzLnJlcXVlc3RQcm9kdWN0RGV0YWlscygxMzMpXHJcbiAgICAgIC50aGVuKGRhdGEgPT4gdGhpcy51cGRhdGVTdGF0ZShkYXRhKSk7XHJcbiAgfVxyXG5cclxuICByZW5kZXIoKSB7XHJcbiAgICByZXR1cm4gKFxyXG4gICAgICA8ZGl2IGNsYXNzTmFtZT17c3R5bGVzLmFwcH0+XHJcbiAgICAgICAgPFNlbGxlckluZm9cclxuICAgICAgICAgIHNlbGxlck5hbWU9e3RoaXMuc3RhdGUuc2VsbGVyTmFtZX1cclxuICAgICAgICAgIGF2ZXJhZ2VTY29yZT17dGhpcy5zdGF0ZS5hdmVyYWdlUmV2aWV3U2NvcmV9XHJcbiAgICAgICAgICBudW1SZXZpZXdzPXt0aGlzLnN0YXRlLm51bWJlclJldmlld3N9XHJcbiAgICAgICAgLz5cclxuICAgICAgICA8SXRlbU5hbWUgaXRlbU5hbWU9e3RoaXMuc3RhdGUuaXRlbU5hbWV9IC8+XHJcbiAgICAgICAgPEJhZGdlIGJhZGdlPXt0aGlzLnN0YXRlLmJhZGdlfSAvPlxyXG4gICAgICAgIDxJdGVtUHJpY2UgaXRlbVByaWNlPXt0aGlzLnN0YXRlLml0ZW1QcmljZX0gLz5cclxuICAgICAgICA8RnJlZVNoaXBwaW5nIGZyZWVTaGlwcGluZz17dGhpcy5zdGF0ZS5mcmVlU2hpcHBpbmd9IC8+XHJcbiAgICAgICAgPFByb2R1Y3RPcHRpb25MaXN0IHByb2R1Y3RPcHRpb25zPXt0aGlzLnN0YXRlLnByb2R1Y3RPcHRpb25zfSB1cGRhdGVDaG9pY2U9e3RoaXMudXBkYXRlUHJvZHVjdENob2ljZX0gLz5cclxuICAgICAgICB7dGhpcy5zdGF0ZS5wZXJzb25hbGl6YXRpb24gJiYgPFBlcnNvbmFsaXphdGlvbiB1cGRhdGVDaG9pY2U9e3RoaXMudXBkYXRlUGVyc29uYWxpemF0aW9uQ2hvaWNlfS8+fVxyXG4gICAgICAgIDxRdWFudGl0eSBhdmFpbGFibGVRdWFudGl0eT17dGhpcy5zdGF0ZS5hdmFpbGFibGVRdWFudGl0eX0gdXBkYXRlUXVhbnRpdHk9e3RoaXMudXBkYXRlUXVhbnRpdHlDaG9pY2V9IC8+XHJcbiAgICAgICAgPEJ1dHRvbnMgLz5cclxuICAgICAgICA8T25PcmRlckF2YWlsYWJsZSBhdmFpbGFibGVRdWFudGl0eT17dGhpcy5zdGF0ZS5hdmFpbGFibGVRdWFudGl0eX0gb25PcmRlcj17dGhpcy5zdGF0ZS5vbk9yZGVyfSAvPlxyXG4gICAgICAgIDxociBjbGFzc05hbWU9e3N0eWxlcy5ib3JkZXJMaW5lfSAvPlxyXG4gICAgICA8L2Rpdj5cclxuICAgIClcclxuICB9XHJcbn1cclxuXHJcbmV4cG9ydCBkZWZhdWx0IEFwcDtcclxuIl0sIm1hcHBpbmdzIjoiOzs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7OztBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7Ozs7O0FBRUE7QUFBQTtBQUNBO0FBREE7QUFDQTtBQUFBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFmQTtBQWlCQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBdkJBO0FBd0JBO0FBQ0E7Ozs7OztBQUNBOzs7Ozs7OztBQUVBO0FBQ0E7O0FBREE7QUFDQTtBQUNBOzs7O0FBQ0E7QUFDQTs7Ozs7Ozs7Ozs7Ozs7Ozs7QUFHQTtBQUNBO0FBQ0E7QUFDQTtBQUNBO0FBQ0E7QUFDQTtBQUZBO0FBSUE7OztBQUVBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFDQTs7O0FBRUE7QUFBQTtBQUNBO0FBQUE7QUFDQTtBQUFBO0FBQ0E7OztBQUVBO0FBQ0E7QUFDQTtBQUFBO0FBRUE7QUFDQTtBQUNBO0FBSEE7QUFLQTtBQUFBO0FBQ0E7QUFBQTtBQUNBO0FBQUE7QUFDQTtBQUFBO0FBQ0E7QUFBQTtBQUFBO0FBQ0E7QUFBQTtBQUNBO0FBQUE7QUFBQTtBQUVBO0FBQUE7QUFBQTtBQUNBO0FBQUE7QUFHQTs7OztBQXBGQTtBQUNBO0FBc0ZBIiwic291cmNlUm9vdCI6IiJ9\n//# sourceURL=webpack-internal:///./client/components/app.js\n");
```

Comment:
You should never commit the contents of `bundle.js`; I suggest removing it from your repo entirely!